css/style.css

```diff
@@ -0,0 +1,353 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    font-size: 15px;
+    font-family:'Roboto', sans-serif;
+    letter-spacing:1.5px;
+    color: #FFFFFF;
+}
+
+.gutter{
+    width:80%;
+    margin: 0 auto;
+}
+
+.brand{
+    color: #FA255E;
+}
+
+h4{
+    font-size: 1.8rem;
+    padding: 20px 0 0 20px;
+}
+
+.split1{
+    background-color: #FA255E;
+    background-image: -webkit-linear-gradient(150deg, #000000 32%, #FA255E 32%);
+}
+
+nav{
+    height:80px;
+    font-size: 1.3rem;
+    padding:10px;
+}
+
+nav a{
+    color: #FFFFFF;
+}
+
+a img{
+    border: 0;
+    /*ie10 fix to remove border from icons*/
+}
+
+.nav-items{
+    list-style: none;
+}
+
+.nav-items li{
+    float: left;
+    margin: 10px;
+    cursor: pointer;
+    font-size: 1.6rem;
+}
+
+.link{
+    text-decoration: none;
+}
+
+.link:hover{
+    text-decoration: underline;
+}
+
+.icons-container{
+    height:20px;
+    width:200px;
+    margin-left: auto;
+}
+
+.icons-container img{
+    float:left;
+    width:50px;
+    margin-left: 15px;
+}
+
+.ham{
+    width:50px;
+    float:left;
+    display:none;
+    margin-top:-5px;
+    margin-left:5px;
+    cursor: pointer;
+}
+
+.headline{
+    text-align:center;
+    font-size:6rem;
+    padding-top:50px;
+    font-family:'Yellowtail', cursive;
+}
+
+.portrait{
+    width:250px;
+    height:250px;
+    border-radius:50%;
+    margin:20px auto;
+    background-image: url("../img/pete.png");
+    background-position:center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border:7px #FFFFFF solid;
+}
+
+.quote{
+    font-size:1.8rem;
+    text-align: center;
+    padding-top:50px;
+    padding-bottom:90px;
+}
+
+.portfolio{
+    background-color:#FFFFFF;
+    height:650px;
+}
+
+.port-container{
+    width:98%;
+    margin:0 auto;
+    padding-top:50px;
+}
+
+.port-area{
+    width:33.3%;
+    float:left;
+}
+
+.port-pic{
+    width:350px;
+    height:350px;
+    border-radius:50%;
+    margin: 10px auto 0 auto;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.port1{
+     background-image: url("../img/pilot.png");
+ }
+
+.port2{
+    background-image: url("../img/cs.jpg");
+}
+
+.port3{
+    background-image: url("../img/cs.jpg");
+}
+
+.hide{
+    display:none;
+}
+
+.port-text{
+    width:80%;
+    margin:5px auto;
+    height:50px;
+    color:#FA255E;
+    text-align:center;
+    padding-top:15px;
+    font-size:1.3rem;
+}
+
+.port-links{
+    width:250px;
+    margin:5px auto;
+}
+
+.demo{
+    float:left;
+    margin: 10px 5px;
+    border-radius:50px;
+    padding:4px 50px;
+    background-color:#FA255E;
+    color: #FFFFFF;
+    font-size:1rem;
+    border:none;
+}
+
+.link-to-code{
+    width:40px;
+    float:right;
+    margin-top:5px;
+}
+
+.about{
+    background-color: #000000;
+    background-image: -webkit-linear-gradient(30deg, #000000 55%, #FA255E 55%);
+    height:650px;
+}
+
+.about-me-area{
```

Comment:
need to add an overflow auto for the floated stuff inside

Reply:
this breaks it so im going to ignore but noted for next time